platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -101,6 +106,13 @@ public void enableLongPress(boolean enable) {
 	public void enablePanningAndScalingGestures(boolean enable) {
 		this.godotGestureHandler.setPanningAndScalingEnabled(enable);
 	}
+	
+	/**
+	 * Set RotaryInput axis. This is Y by default.
```

Comment:
This javadoc should be updated to match the added documentation.

Reply:
Fixed.